cpp/change-notes/2021-12-14-overruning-write-split.md

```diff
@@ -0,0 +1,2 @@
+lgtm,codescanning
+* A new `cpp/very-likely-overruning-write` query has been added for C/C++, which reports results formerly flagged by `cpp/overruning-write` where a non-trivial range analysis shows a higher likelyhood of an actual problem. As a consequence, the new query is tagged with high precision.
```

Comment:
I think this is maybe too technical for a change-note. How about this instead? (I've also suggested a note about the query being enabled by default.)

```suggestion
* A new `cpp/very-likely-overruning-write` query has been added to the default query suite for C/C++. The query reports some results that were formerly flagged by `cpp/overruning-write`.
```